code/ui/blocks/src/examples/Button.stories.tsx

```diff
@@ -84,11 +84,22 @@ export const Clicking: Story = {
   },
 };
 
-export const ClickingInDocs: Story = {
+export const ClickingInDocsDeprecated: Story = {
```

Comment:
do we need both? wasn't this introduced in 7.0 alpha? If so, I don't think we need to deprecate it, just remove it.